### DISCUSSION TITLE
404 div when clicking on FileManager menu

### DISCUSSION THREAD

**Discussion by ymauron** [oneduo/nova-file-manager#7] [General]
Hi, 

I really like the idea of this package. I installed it but get a 404 error when clicking on the FileManager menu. Any idea on how to debug this ? 

Thanks

**Comment by mikaelpopowicz** [comment]
Hi @ymauron 

Did you add the tool in your NovaServiceProvider ? Did you clear your route cache ?

Some other issues were reported, we are still working on.

**Reply by ymauron** [reply]
Hi @mikaelpopowicz, 

Thanks for the reply. Yes, on both. The tool has been added and I cleared the route. I also check and the route is appearing however with a different base :

  GET|HEAD        nova-vendor/nova-file-manager ........................................................................ nova-file-manager.data › BBSLab\NovaFileManager › IndexController

**Reply by PeterOpal** [reply]
Than did you solve this problem?
